### DISCUSSION TITLE
Interested in the Smart Meeting Notes project

### DISCUSSION THREAD

**Discussion by prarabdhshukla** [openvinotoolkit/openvino#16693] [Google Summer of Code]
Hi,
I am Prarabdh Shukla, a third year BTech Honours student in Computer Vision at the Indian Institute of Technology Bhilai. My honours thesis work is in Machine Learning. One track of my research is on Federated Learning and the other is on Algorithms for High Dimensional Data. I am comfortable with PyTorch as well as Tensorflow due to extensive use of both in my advanced ML courses and research work. I have a decent development experience too thanks to my work with the Election Commission of India and Edliy. You can find my resume [here](https://drive.google.com/file/d/1_8dniRXxE_O6rwC56JGsM83oYmhJu9pY/view?usp=sharing). 

I was interested in the Smart Meeting Notes project. The high level description of my idea is as follows: 
We can easily obtain the audio and video of any meeting. Using the time sync between the two, we can determine correlation between parts of the audio and the video (i.e., what part of the video is actually related to what part of the audio). Now, we can use a summarizer on the audio transcript and also use the correlation we obtained with the video to add pictures and nice visuals to our notes. The novelty here is that rather than using a simple audio only approach, we can indeed make our notes 'smart' by adding pictures and visualizations(maybe ones from the internet too which were not a part of the meeting's video). 

I can write this down in the form of a proposal and share it with you shortly. Please let me know if there are some inputs from your end. For the model implementation task, I am working on that as well, although, I might make a PR a little later than 4th April. Please let me know if this works or do I have finish the PR before submitting the proposal. 

Thanks and Regards, 
Prarabdh

**Comment by adrianboguszewski** [comment]
Hi @prarabdhshukla, the link to your PR must be included in your proposal and the application must be submitted before GSoC due date. Please note we cannot accept any application without a PR link.
